community-content/sklearn_text_classification_from_script_using_vertex_sdk/create_job.ipynb

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
<p>The container images appears to be hardwired to "europe". You can derive this by the setting of LOCATION.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Addressed by setting 
```python
LOCATION = f"{REGION}-{ZONE}"`
IMAGE = f"{REGION}-docker.pkg.dev/vertex-ai/training/scikit-learn-cpu.0-23:latest"
```

However, I think the Python client could set this automatically to the closest region, similar to how it's done in the Cloud Console. [From the docs](https://cloud.google.com/vertex-ai/docs/training/pre-built-containers):

> If you use the Google Cloud Console to perform custom training, the Cloud Console selects the URI that best matches the location where you are using Vertex AI in order to reduce latency.